<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortCircuit</class>
 <widget class="QWidget" name="ShortCircuit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>数据测试</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="ShortTest">
     <property name="text">
      <string>短路数据</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="OpenTest">
     <property name="text">
      <string>开路数据</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="StartTest">
     <property name="text">
      <string>开始测试</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="OutData">
     <property name="text">
      <string>导出数据</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="CalibrationData">
     <property name="text">
      <string>校验数据</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabStopWidth">
      <number>80</number>
     </property>
     <property name="cursorWidth">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="FlatnessTest">
     <property name="text">
      <string>平整度数据</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="InData">
     <property name="text">
      <string>导入数据</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>1</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="10">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="9" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="Enlarge">
     <property name="text">
      <string>放大</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
